project/profiles/capacity4more/modules/c4m/features/c4m_features_overview_groups/c4m_features_overview_groups.module

```diff
@@ -23,3 +23,12 @@ function c4m_features_overview_groups_c4m_search_page_info() {
     ),
   );
 }
+
+/**
+ * Implements hook_views_data_alter().
+ *
+ * Custom handler for the title field. Handles "Disable PURL" option.
+ */
+function c4m_features_overview_groups_views_data_alter(&$data) {
+  $data['node']['title']['field']['handler'] = 'c4m_overview_groups_handler_title_no_purl';
```

Comment:
@ordavidil it's a string -
![screen shot 2016-08-10 at 2 16 13 pm](https://cloud.githubusercontent.com/assets/13344923/17551843/09b2ebac-5f05-11e6-86b8-25fb924ecc18.png)